src/Microsoft.IdentityModel.Logging/Microsoft.IdentityModel.Logging.csproj

```diff
@@ -12,6 +12,21 @@
     <PackageTags>.NET;Windows;Authentication;Identity;Logging</PackageTags>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
```

Comment:
we can have a single group for debug, we don't need a separate group for the different targets.

Reply:
fixed